library/packages/src/lib/y2packager/licenses_fetchers/rpm.rb

```diff
@@ -23,7 +23,7 @@ class Rpm < Base
       #
       # @return [String, nil] Product's license; nil if the product or the license were not found
       def content(lang)
-        super
+        return @default_content if default_lang?(lang) && @default_content
```

Comment:
I am not sure if we understand each other. Super was fine for me, just here you ignore its output so simple change can be
```
result = super
return result if result
```

Reply:
Ouch! I did it in a first attempt to accomplish your suggestion... but it seems a bit weird to me. Let me to bring it back.

Reply:
After test a bit more this, I think we could keep this logic here. Moving it again to the base class and using those two additional lines in each child seems weird to me. Too much indirection, isn't it?

Reply:
fine for me.